test/integration/targets/ios_linkagg/tests/cli/basic.yaml

```diff
@@ -0,0 +1,181 @@
+#---
+#- debug: msg="START cli/basic.yaml on connection={{ ansible_connection }}"
```

Comment:
Any specific reason for commenting out the test cases?

Reply:
We do not have l2 switch ios image for dci test.

Reply:
Can when condition be added in task instead of commenting the testcases.